delta/utils/solver/asr_solver.py

```diff
@@ -181,16 +184,25 @@ def model_fn(self, mode):
       # data must be (features, labels), only using features as input
       self.model.build(input_shape=self.batch_input_shape[0])
 
-    # parallel and compile model
-    self.build(multi_gpu=(mode == utils.TRAIN))
-
-    if mode != utils.TRAIN:
-      model_path = Path(self._model_path).joinpath('best_model.ckpt')
+    assert self._model_load_type in ['best', 'lastest', 'no']
+    assert self._init_epoch in range(0, self._num_epochs)
+    if mode != utils.TRAIN or self._model_load_type != 'no':
+      model_path = None
+      if self._model_load_type == "best":
+        model_path = Path(self._model_path).joinpath('best_model.ckpt')
+      elif self._model_load_type == 'lastest':
+        file_name_pattern = 'model.{epoch:02d}-{monitor_used:.2f}.ckpt'
```

Comment:
extract `'model.{epoch:02d}-{monitor_used:.2f}.ckpt'`  as a variable since it is used in ModelCheckpointCallbacks to save ckpt.